pkg/config/config.go

```diff
@@ -20,19 +20,22 @@ package config
 import (
 	"io/ioutil"
 	"fmt"
+	"os"
 
 	"github.com/BurntSushi/toml"
 )
 
 const (
-	configDefault = "./res/configuration.toml"
-	configDocker = "./res/configuration-docker.toml"
-	configUnitTest = "./res/configuration-test.toml"
+	configDirectory = "./res/"
+	configDefault = "configuration.toml"
+	configDocker = "configuration-docker.toml"
+	configUnitTest = "configuration-test.toml"
 )
 
 func LoadFromFile(profile string, configuration interface{}) error {
-	path := determineConfigFile(profile)
-	contents, err := ioutil.ReadFile(path)
+	path := determinePath()
+	fileName := determineConfigFile(profile)
+	contents, err := ioutil.ReadFile(path + fileName)
```

Comment:
I think it would be more self-documenting if we were not to force `path` ending with `/`, but rather have here something like: `path + "/" + filename`

Reply:
Good, this is something I wanted to get feedback on. I don't have a strong preference.